src/object.rs

```diff
@@ -1291,6 +1293,10 @@ pub trait ObjectExt: ObjectType {
     fn find_property<'a, N: Into<&'a str>>(&self, property_name: N) -> Option<::ParamSpec>;
     fn list_properties(&self) -> Vec<::ParamSpec>;
 
+    fn set_qdata(&self, key: Quark, value: Box<dyn Any>);
+    fn get_qdata(&self, key: Quark) -> Option<&dyn Any>;
+    fn steal_qdata(&self, key: Quark) -> Option<Box<dyn Any>>;
```

Comment:
This has to be `unsafe` unfortunately as other code might store arbitrary pointers at the same quark and then everything would fall apart. That's the main reason why I didn't add this API yet :)

Reply:
Marking these functions as `unsafe` makes this PR pointless. :(
In this case I prefer even [less safe API](https://github.com/gtk-rs/glib/pull/644) but at least it is zero-cost (this one has two nested `Box`es).

Reply:
Indeed, let's go with that then :) Sorry for the trouble